### PR TITLE
Add documentation on how to use age in .sops.yaml

### PR DESCRIPTION
**Pull request by hramberger**
Add the option to use age via the .sops.yaml and updated descriptions to match the given code example.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -617,23 +617,25 @@ can manage the three sets of configurations for the three types of files:
 	# creation rules are evaluated sequentially, the first match wins
 	creation_rules:
 		# upon creation of a file that matches the pattern *.dev.yaml,
-		# KMS set A is used
+		# KMS set A as well as PGP and age is used
 		- path_regex: \.dev\.yaml$
 		  kms: 'arn:aws:kms:us-west-2:927034868273:key/fe86dd69-4132-404c-ab86-4269956b4500,arn:aws:kms:us-west-2:361527076523:key/5052f06a-5d3f-489e-b86c-57201e06f31e+arn:aws:iam::361527076523:role/hiera-sops-prod'
 		  pgp: 'FBC7B9E2A4F9289AC0C1D4843D16CEE4A27381B4'
+      age: 'age129h70qwx39k7h5x6l9hg566nwm53527zvamre8vep9e3plsm44uqgy8gla'
```
Comment (inverse) [inline README.rst:624]: This indentation seems off to me - does it need to be indented more?

File: README.rst [hunk 2]
```diff
@@ -617,23 +617,25 @@ can manage the three sets of configurations for the three types of files:
...
 
-		# prod files use KMS set B in the PROD IAM
+		# prod files use KMS set B in the PROD IAM, PGP and age
 		- path_regex: \.prod\.yaml$
 		  kms: 'arn:aws:kms:us-west-2:361527076523:key/5052f06a-5d3f-489e-b86c-57201e06f31e+arn:aws:iam::361527076523:role/hiera-sops-prod,arn:aws:kms:eu-central-1:361527076523:key/cb1fab90-8d17-42a1-a9d8-334968904f94+arn:aws:iam::361527076523:role/hiera-sops-prod'
 		  pgp: 'FBC7B9E2A4F9289AC0C1D4843D16CEE4A27381B4'
+      age: 'age129h70qwx39k7h5x6l9hg566nwm53527zvamre8vep9e3plsm44uqgy8gla'
```
Comment (inverse) [inline README.rst:630]: Same with this one

**Review by inverse — COMMENTED** [review]
Aside the comments LGTM :+1:

**Review by inverse — APPROVED** [review]

**Review by hiddeco — APPROVED** [review]
This looks good to me as well :bow: 

Can you please smash your commits together, rebase, and sign-off on the commit?

File: README.rst [hunk 3]
```diff
@@ -594,19 +594,19 @@ KMS and PGP master keys defined in the file.
 
 	sops -r example.yaml
 
-Using .sops.yaml conf to select KMS/PGP for new files
+Using .sops.yaml conf to select KMS/PGP/AGE for new files
```
Comment (hiddeco) [inline README.rst:597]:
```suggestion
Using .sops.yaml conf to select KMS, PGP and age for new files
```

**Comment by hiddeco** [comment]
Perfect, thank you very much 🙇🏼‍♂️

**Pull request merged**